iml-gui/crate/src/components/alert_indicator.rs

```diff
@@ -56,13 +48,40 @@ pub(crate) fn alert_indicator<T>(
 
     let icon = font_awesome_outline(class![cls, C.inline, C.h_4, C.w_4], "bell");
 
-    if compact {
+    let count = alerts.len();
+
+    let txt = if count == 0 {
+        "No Alerts".into()
+    } else if count == 1 {
+        alerts[0].message.clone()
+    } else {
+        format!("{} Alerts", alerts.len())
+    };
+
+    let mut el = if compact {
         span![
             class![C.inline_block],
-            span![attrs::container(), icon, tooltip::view(&msg, tt_placement)],
+            attrs::container(),
+            icon,
+            tooltip::view(&txt, tt_placement),
             sup![class![cls], count.to_string()]
         ]
     } else {
-        span![icon, span![class![C.ml_1], &msg]]
+        span![icon, attrs::container(), span![class![C.ml_1], &txt]]
+    };
+
+    if count > 1 {
+        let pop = popover::view(
+            popover::content_view(ul![
+                class![C.list_disc, C.px_4, C.whitespace_no_wrap, C.text_left],
+                alerts.iter().map(|a| li![a.message.clone()]).collect::<Vec<_>>()
```

Comment:
```suggestion
                alerts.into_iter().map(|a| li![&a.message])
```

Both the clone and collect are not needed.